subsys/net/ip/net_pkt.c

```diff
@@ -888,6 +888,23 @@ void net_pkt_get_info(struct k_mem_slab **pkts,
 	}
 }
 
+struct net_pkt *net_pkt_pullup(struct net_pkt *pkt, int len)
```

Comment:
Let's keep the same return type as all net_pkt_* function, a int.

And I think it's better not to unref pkt here, let this function do one thing (verify contiguity and set the iter at right place) and not mangle with memory.

Reply:
> Let's keep the same return type as all net_pkt_* function, a int.

No, let's keep it. Please don't insist on this. Make the "int" version and compare the amounrt of housekeeping code in typical use.